docs/ref/contrib/sitemaps.txt

```diff
@@ -483,11 +483,18 @@ that: :func:`django.contrib.sitemaps.ping_google()`.
 
 .. function:: ping_google
 
-    :func:`ping_google` takes an optional argument, ``sitemap_url``,
-    which should be the absolute path to your site's sitemap (e.g.,
-    :file:`'/sitemap.xml'`). If this argument isn't provided,
-    :func:`ping_google` will attempt to figure out your
-    sitemap by performing a reverse looking in your URLconf.
+    :func:`ping_google` takes the following optional arguments:
+
+        * ``sitemap_url`` which should be the absolute path to your site's sitemap (e.g.,
+          :file:`'/sitemap.xml'`). If this argument isn't provided,
+          :func:`ping_google` will attempt to figure out your
+          sitemap by performing a reverse looking in your URLconf.
+
+        * ``-d --site_domain``, If you are not using ``django.contrib.sites``, use this
+          to pass your site's domain (e.g., ``example.com``)
+
+        * ``-s --is_secure``, A boolean to indicate to use ``https://`` scheme instead of
+          ``http://``. Defaults to ``False``.
```

Comment:
you need the `versionadded` annotation for the flags added in django 2.2

Reply:
wait, you've documented the flags under the function documentation. This should really be about `site_domain` not `-d --site_domain`. The flags should be documented in the last section, "Pinging Google via manage.py".